src/applications/vaos/components/calendar/CalendarWidget.jsx

```diff
@@ -105,85 +103,27 @@ export default class CalendarWidget extends Component {
     this.setState({ months: updatedMonths });
   };
 
-  isValid = date => {
-    let isValid = true;
-    const { additionalOptions } = this.props.additionalOptions;
-
-    if (additionalOptions) {
-      if (
-        additionalOptions.required &&
-        isDateInSelectedArray(date, this.state.selectedDates)
-      ) {
-        isValid = false;
-        document.getElementById(`date-cell-${date}`).focus();
-        this.setState({ optionsError: additionalOptions.validationMessage });
-      } else {
-        this.setState({ optionsError: null });
-      }
-    }
-
-    return isValid;
-  };
```

Comment:
Removed this since we are performing validations on the page itself, rather than within the `CalendarWidget`